### PR TITLE
[documentation] Add local CI troubleshooting for readiness + agent artifacts

### LINKED ISSUE #643: [documentation] Add local CI troubleshooting for readiness + agent artifacts
## Why / Context
CI/readiness troubleshooting steps are currently spread across output messages and docs. A focused troubleshooting section would reduce time-to-recovery when local CI fails on artifact freshness or readiness inputs.

## Scope
**In scope**
- Document a concise troubleshooting flow for `agent-regenerate --verify` failures.
- Document readiness gate prerequisites (report freshness, profile paths, expected files).
- Cross-link local CI usage docs to this troubleshooting guidance.

**Out of scope**
- Broad docs reorganization.
- Changing production readiness policy.

Constraints:
- Keep changes small/mergeable.
- Deterministic tests only.
- Avoid touching unrelated generated artifacts unless required.

## Acceptance Criteria (required)
- [ ] Docs include step-by-step recovery for stale artifact failures.
- [ ] Docs include readiness gate input prerequisites and stale-data interpretation.
- [ ] Local CI docs link directly to troubleshooting section.
- [ ] Docs lint/audit checks pass.

## Implementation Notes / Pointers
**Likely files / modules:**
- `docs/DEVELOPMENT_GUIDELINES.md`
- `docs/READINESS.md`
- `AGENTS.md`

**Related tests:**
- `scripts/maintenance/docs_link_audit.py`
- `tests/unit/scripts/test_docs_maintenance.py`

**Edge cases to handle:**
- Ensure commands match current CLI entrypoints.
- Keep guidance profile-aware (`canary` vs others).
- Avoid stale examples with outdated paths.

## Commands (local)
- `make fmt`
- `make lint`
- `uv run pytest -q tests/unit/scripts/test_docs_maintenance.py`
- `python scripts/maintenance/docs_link_audit.py`

## PR Requirements
- PR title should match the issue.
- PR body must include: `Fixes #<issue-number>`
- CI must be green; if Agent Artifacts Freshness fails, run `uv run agent-regenerate`, then commit `var/agents/...` updates.

## Codex-Ready Checklist (for the issue creator)
- [x] Clear acceptance criteria
- [x] At least one file pointer
- [x] Commands included
- [x] No ambiguous “do the right thing” language

### PR DESCRIPTION
**Pull request by Solders-Girdles**
Fixes #643

(Automated PR created by OpenClaw PR finisher)
Branch: codex/643-documentation-add-local-ci-troubleshooti
Commit: ef10e251aa484db3c0778d716c5e2169ae6888e8

### CODE REVIEW

File: docs/READINESS.md [hunk 1]
```diff
@@ -253,3 +253,16 @@ Notes:
...
+
+The gate follows the `PREFLIGHT_PROFILE`/`READINESS_REPORT_DIR` pair, so swap `canary` for `prod`, `dev`, or another profile whenever you run the gate locally. For a detailed troubleshooting flow that connects `uv run local-ci` to these inputs, see the [Local CI troubleshooting](docs/DEVELOPMENT_GUIDELINES.md#local-ci-troubleshooting) steps in the development guidelines.
```
[the remark this answers is not in the document]
Comment (Solders-Girdles) [reply docs/READINESS.md]: False positive — same issue as above. The actual link is:

```markdown
[Local CI troubleshooting](DEVELOPMENT_GUIDELINES.md#local-ci-troubleshooting)
```

The target is `DEVELOPMENT_GUIDELINES.md`, not `docs/DEVELOPMENT_GUIDELINES.md`. Both files are in `docs/`, so the relative path resolves correctly to `docs/DEVELOPMENT_GUIDELINES.md`.

File: docs/DEVELOPMENT_GUIDELINES.md [hunk 2]
```diff
@@ -137,6 +137,27 @@ If you prefer calling the script directly, run:
...
+3. If the gate complains about report age, regenerate the report and optionally raise `GPT_TRADER_READINESS_MAX_REPORT_AGE_DAYS` (or pass `--max-report-age-days`) when your cadence is longer than the default 7 days; add `--strict` or set `GPT_TRADER_READINESS_STRICT=1` if you want the gate to fail instead of degrade.
+4. Rerun `uv run python scripts/ci/check_readiness_gate.py --profile <profile>` or `uv run local-ci` to confirm the gate now sees the refreshed data.
+5. The gate also reads `runtime_data/<profile>/events.db` for liveness and `var/data/status.json` (or your configured status file), so ensure those files exist alongside the report directory before rerunning local CI.
+6. For more background on the required files, freshness windows, and how stale data is interpreted, see [Readiness gate inputs & stale-data interpretation](docs/READINESS.md#readiness-gate-inputs--stale-data-interpretation).
```
[the remark this answers is not in the document]
Comment (Solders-Girdles) [reply docs/DEVELOPMENT_GUIDELINES.md]: Correction to my earlier reply: the Codex finding was valid at the time of commit ef10e251aa — that commit did use `docs/READINESS.md`. This was already fixed in the follow-up commit 798ea706 which corrected the cross-links. The conversation can be resolved.

File: docs/READINESS.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Solders-Girdles) [reply docs/READINESS.md]: Correction: same as above — the original commit did use `docs/DEVELOPMENT_GUIDELINES.md`, and commit 798ea706 already corrected it. This conversation can be resolved.
